GPU/Common/PresentationCommon.cpp

```diff
@@ -437,7 +438,8 @@ void PresentationCommon::CopyToOutput(OutputFlags flags, int uvRotation, float u
 	// Make sure Direct3D 11 clears state, since we set shaders outside Draw.
 	draw_->BindPipeline(nullptr);
 
-	int indexes[] = { 0, 1, 2, 0, 2, 3 };
+	// Wouldn't it be enough to do this only when creating the index buffer?
```

Comment:
Yeah, we should probably move it there and just do it once.  I copied from softgpu and wasn't sure if I wanted to make it use a strip/fan.

-[Unknown]

Reply:
Fans are a bit poorly supported in APIs so I'd stay away from those, but a strip should work, right? Either way, I'll do that separately, want to get this in separately.